.github/workflows/GuideConverterAction.yaml

```
@@ -0,0 +1,100 @@
+name: GuideConverter
+on: 
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: 'Branch pushed to'     
+        required: true
+        default: 'defaulted'
+      guide_name:
+        description: 'Guide changed'
+        required: true
+        default: 'not given'
+        
+jobs:
+  printBranch:
+    runs-on: ubuntu-latest
+
+    steps: 
+
+      - name: Output message
+        run: |
+          echo "${{ github.event.inputs.branch }}"
+          echo "${{ github.event.inputs.guide_name }}"
+  convertMaster:
+    name: Make PR to Master
+    runs-on: ubuntu-latest
+    if: contains(github.event.inputs.branch, 'sn-automation-testing-master')
+  
+    steps:
+
+      # Any prerequisite steps
+      - uses: actions/checkout@master
+
+      # Print message
+      - name: Message
+        run: |
+          echo "${{ github.event.inputs.branch }}"
+          echo "${{ github.event.inputs.guide_name }}"
+      - name: Run Converter
+        run: |
+          git init
+          mkdir -p instructions/${{ github.event.inputs.guide_name }}/
+          rm -f instructions/${{ github.event.inputs.guide_name }}/README.md
+          javac GuideConverterV2.java
+          java GuideConverterV2 ${{ github.event.inputs.guide_name }} master
+          rm -f importFunctions.class
+          rm -f functions.class
+          rm -f GuideConverterV2.class
+          mv ${{ github.event.inputs.guide_name }}.md instructions/${{ github.event.inputs.guide_name }}/instructions.md
+          git add .
+          git config --global user.email "GuidesBot@OpenLiberty.io"
+          git config --global user.name "GuidesBot"
+          git commit -m "Updated by github actions from ${{ github.event.inputs.guide_name }}"
+    
+      - uses: peter-evans/create-pull-request@v3
+        with:
+            title: "Change to ${{ github.event.inputs.guide_name }}"
+            body: "Updated by github actions, triggered by the ${{ github.event.inputs.guide_name }} repo."
+            branch: "${{ github.event.inputs.guide_name }}/actions"
```

```suggestion
            branch: "${{ github.event.inputs.guide_name }}/master"
```

Minor change to separate PR's from master/qa branches.

---

Done